modules/tinymce/src/core/main/ts/caret/CaretUtils.ts

```diff
@@ -44,13 +44,13 @@ const findNode = (node: Node, direction: number, predicateFn: (node: Node) => bo
 
   if (isBackwards(direction)) {
     if (isCefOrCaretContainer) {
-      node = skipCaretContainers(walker.prev, true);
+      node = skipCaretContainers(walker.prev.bind(walker), true);
```

Comment:
Did this function before? And if so how

Reply:
It works because it's actually bound within the class itself as well. See: https://github.com/tinymce/tinymce/blob/7e4aa1b65df2c4cd299c51dd177fead8d3a602e2/modules/tinymce/src/core/main/ts/api/dom/TreeWalker.ts#L34 So it worked because of that and this isn't technically needed, but it'd be good to remove that one day.

The other thing is that we aren't guaranteed someone else hasn't done `TreeWalker.prototype.prev = () => 'do something else'` here, so this does add some level of additional safety.